mobile_app_connector/models/recurring_contract.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+import logging
+
+from odoo import models, api
+from ..mappings.compassion_child_mapping import MobileChildMapping
+
+
+logger = logging.getLogger(__name__)
+
+
+class CompassionChild(models.Model):
+    """ A sponsored child """
+    _inherit = 'recurring.contract'
+
+
+    @api.multi
+    def get_app_json(self, multi=False):
+        """
+        Called by HUB when data is needed for a tile
```

Comment:
Maybe you could explain here why you need the same child data from the contract object?